core/src/main/scala/org/bitcoins/core/crypto/ExtKey.scala

```diff
@@ -171,6 +175,19 @@ object ExtPrivateKey extends Factory[ExtPrivateKey] {
             "Fingerprint must be 4 bytes in size, got: " + fingerprint)
   }
 
+  def fromString(hex: String): Try[ExtPrivateKey] =
```

Comment:
Why do we have this here? Is this just a safe version of `fromHex`?

Reply:
1) variable name is confusing from by side, should be `base58` and not `hex`. 
2) method already exists on `ExtKey`, but the user has to perform a match/cast to get a private/public key. This (and the correpsonding method in `ExtPublicKey`) does that match/cast for them

Reply:
1) variable name is confusing from by side, should be `base58` and not `hex`. 
2) method already exists on `ExtKey`, but the user has to perform a match/cast to get a private/public key. This (and the corresponding method in `ExtPublicKey`) does that match/cast for them